app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -92,8 +94,8 @@ private boolean isCandyWordFirst() {
      * @param simpleBG     Whether the bitmap has a simple background
      * @return Bitmap with replaced colors
      */
-    private Bitmap replaceColors(Bitmap srcBitmap, int keepCr, int keepCg, int keepCb, int replaceColor, int distance,
-                                 boolean simpleBG) {
+    private Bitmap replaceColors(Bitmap srcBitmap, Bitmap dstBitmap, int keepCr, int keepCg, int keepCb,
```

Comment:
I'm overly sleepy but I just added the new API in (untested) 810873854f4e20192119183474bd58b452c8bb54. I can squash with the previous commit.